File: 287. Find the Duplicate Number/solution.go
func findDuplicate(nums []int) int {
    n := len(nums)
    for i := 1; i < n+1; i++ {
        count := 0
        for j := 0; j < n; j++ {
            if nums[j] == i {
                count++
            }
        }
        if count > 1 {
            return i
        }
    }
    return 1
}

File: 287. Find the Duplicate Number/solution.java
class Solution {
    public int findDuplicate(int[] nums) {
        int n = nums.length;
        int num;
        for (int i = 0; i < n; i++) {
            num = Math.abs(nums[i]);
            if (nums[num-1] < 0) {
                return num;
            } else {
                nums[num-1] *= -1;
            }
        }
        return nums[n-1];
    }
}
